src/http/client.js

```diff
@@ -42,4 +72,25 @@ export default class HttpClient {
 
 		return `?${queries.join('&')}`;
 	}
+
+	/**
+	 * Build a json body with the object provided
+	 * @param {object} object
+	 * @return {object}
+	 */
+	buildJsonBody(object) {
+		return JSON.stringify(object);
+	}
+
+	/**
+	 * Buid a form data body with the object provided
+	 * @param {object} object
+	 * @return {object}
+	 */
+	buildFormDataBody(object) {
+		return Object.keys(object).reduce((formData, key) => {
```

Comment:
😂

Reply:
![all the things](https://i.imgflip.com/2cfk9p.jpg "All the things")